_adr/76/index.adoc

```diff
@@ -0,0 +1,96 @@
+---
+num: 76
+title: "Handling usage of dynamically-async-created DNS records from dependent services"
+status: "Accepted"
+authors:
+  - "Gerard Ryan"
+tags:
+  - kafka
+applies_padrs: # What PADRs does this ADR apply?
+applies_patterns: # What APs does this ADR apply?
+---
+
+:link-adr-57: link:../57/index.html[ADR-57]
+:abbr-rhosak: pass:[<abbr title="Red Hat OpenShift Streams for Apache Kafka">RHOSAK</abbr>]
+// Top style tips:
+// * Use one sentence per line
+// * No unexpanded acronyms
+// * No undefined jargon
+
+// No need for a title heading, it's added by the template
+
+== Context and Problem Statement
```

Comment:
Deviated from [template](https://github.com/bf2fc6cc711aee1a0c2a/architecture/blob/main/_adr/0/index.adoc) slightly here for headings, to use the more Asciidoc notation of `==` instead of `##` (I think the latter works because of some Asciidoctor compatibility for that Markdown syntax—a different processor might not). I can create a PR to update the template if that would be useful too.

Reply:
A PR to update the template would be great. Thanks!